powerstore/provider.go

```diff
@@ -185,6 +187,7 @@ func (p *Pstoreprovider) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Diag
 // this helper can be skipped and the provider type can be directly type
 // asserted (e.g. provider: in.(*provider)), however using this can prevent
 // potential panics.
+//
 //lint:ignore U1000 used by the internal provider, to be checked
```

Comment:
if comment is not required, please remove it

Reply:
this new line is result of gofmt

Reply:
I am talking about this line "lint:ignore U1000 used by the internal provider, to be checked"